src/coreComponents/python/CMakeLists.txt

```diff
@@ -29,11 +34,7 @@ set( GEOSX_XML_TOOLS_BINS
 
 
 add_custom_command( OUTPUT ${GEOSX_XML_TOOLS_BINS}
-                    COMMAND bash ${CMAKE_SOURCE_DIR}/../scripts/setupVirtualPythonEnvironment.bash -p ${PYTHON_POST_EXECUTABLE} -o ${CMAKE_BINARY_DIR}/python
-                    COMMAND ln -s ${CMAKE_BINARY_DIR}/python/geosx/bin/preprocess_xml ${CMAKE_BINARY_DIR}/bin/preprocess_xml
-                    COMMAND ln -s ${CMAKE_BINARY_DIR}/python/geosx/bin/format_xml ${CMAKE_BINARY_DIR}/bin/format_xml
-                    COMMAND ln -s ${CMAKE_BINARY_DIR}/python/geosx/bin/python ${CMAKE_BINARY_DIR}/bin/python
-                    COMMAND ln -s ${CMAKE_SOURCE_DIR}/../scripts/automatic_xml_preprocess.sh ${CMAKE_BINARY_DIR}/bin/geosx_preprocessed
+                    COMMAND bash ${CMAKE_SOURCE_DIR}/../scripts/setupPythonEnvironment.bash -p ${PYTHON_POST_EXECUTABLE} -b ${CMAKE_BINARY_DIR}/bin
```

Comment:
Since the setup script is accepting `-m` and `-v` options, should there be a way for a user to pass them through a host-config?

Reply:
Sure, but since we are moving into a default-pygeosx direction, which is already virtual, these would be largely obsolete.